examples/data_preprocess/coder1.py

```diff
@@ -0,0 +1,314 @@
+# SPDX-FileCopyrightText: (c) iSE UIUC Research Group
```

Comment:
please update license according to `tests/sanity/check_license.py` (or add custom headers)